src/NosCore.GameObject/Networking/LoginService/LoginService.cs

```diff
@@ -63,17 +64,17 @@ public class LoginService : ILoginService
             {
                 clientSession.SessionId = clientSession.Channel?.Id != null
                     ? SessionFactory.Instance.Sessions[clientSession.Channel.Id.AsLongText()].SessionId : 0;
-/*
-                if (false) //TODO Maintenance
-                {
-                    await clientSession.SendPacket(new FailcPacket
-                    {
-                        Type = LoginFailType.Maintenance
-                    });
-                    await clientSession.Disconnect();
-                    return;
-                }
-*/
+                /*
+                                if (false) //TODO Maintenance
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5326532)